test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +42,10 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    @Test
+    public void isAnyNull() throws Exception {
+        assertTrue(Utils.isAnyNull(Integer.valueOf(1), null));
+        assertTrue(Utils.isAnyNull("a", "b", null));
```

Comment:
It would be good if you also tested the False case and combining multiple datatypes as discussed in tutorial.